package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "math-magicians-react",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
- [ ] ESLint is not properly installed. Kindly ensure to setup ESLint using the guidelines provided in [this link](https://github.com/microverseinc/linters-config/tree/master/react-redux)

Reply:
Hello Robert,
Thanks for the review. I already submitted the project after this which is dependent on this one and there was no issue of sorts.